### ISSUE TITLE
Arrow keys not breaking out of text selection

### ISSUE THREAD

**Issue by Stanzilla** [kvirc/KVIrc#1738]
Repro: Write a longer sentence, hit ctrl a to mark it all and then try to use the arrow keys to deselect and jump to the end of it

Pretty sure that used to work before but @wodim says he encountered it ages ago. Not sure what's up but @staticfox probably knows since he touched the input box code last.

```
KVIrc 4.3.2 'Aria'

Runtime Info:
System Name: Windows Release : 6.2.9200
System Version: Windows 8 Professional with Media Center (x64) (Build 9200)
Architecture: Intel64 Family 6 Model 58 Stepping 9

Build Info:
Build Date: 2015-11-04 22:27:58 UTC
Sources Date: 20151104
Revision Number: git-5656-g68ef422
System Name: Windows-6.2
CPU Name: AMD64
Build Command: C:/Program Files (x86)/CMake/bin/cmake.exe
Build Flags: 
   MANDIR=share/man
   CMAKE_INSTALL_PREFIX=C:/projects/kvirc/build/release
   Threads=Win32
Compiler Name: C:/Program Files (x86)/Microsoft Visual Studio 12.0/VC/bin/amd64/cl.exe
Compiler Flags: /DWIN32 /D_WINDOWS /W3 /GR /EHsc
```

**Comment by un1versal** [comment]
Which OS, which KVIrc version?

Also its extra incomplete because personally I have no idea what I should be looking for, what the issue is exactly and whats the expected correct behavior.

:-1: to this report

**Comment by Stanzilla** [comment]
:-1: to your comment

**Comment by Stanzilla** [comment]
Just read the text, it has the steps to repro...

**Comment by un1versal** [comment]
> Also its extra incomplete because personally **I have no idea what I should be looking for, what the issue is exactly and whats the expected correct behavior.**

**Comment by Stanzilla** [comment]
>Arrow keys not breaking out of text selection
>Write a longer sentence, hit ctrl a to mark it all and then try to use the arrow keys to deselect and jump to the end of it

**Comment by Stanzilla** [comment]
Let me rephrase then:
1. Go to the input box of the KVIrc main window on a random channel
2. Write a longer (optional step) sentence, hit <kbd>CTRL</kbd>/<kbd>⌘</kbd> + <kbd>A</kbd> to mark it all and then try to use the <kbd>→</kbd> _and/or_ <kbd>←</kbd> keys

What it should do: Deselect the current selection
What it does: Not deselect the current selection

I hope I could clear that up for you (and others).

**Comment by wodim** [comment]
Very bad description of the problem; Mac uses <kbd>⌘</kbd> instead of <kbd>CTRL</kbd>. Don’t make me remove your comment, you fucking retard.

:-1: To you and your issue.

**Comment by Stanzilla** [comment]
Okay sorry, edited.
